docs/design/ovs-pipeline.md

```diff
@@ -1078,15 +1093,19 @@ Some bits of ct mark are persisted:
 - The value of `PktSourceField` is persisted to `ConnSourceCTMarkField`, storing the source of the connection for the
   current packet and subsequent packets of the connection.
 
+Flow 4 is to resubmit the packets which are not matched by flows 1-3 back to table [ServiceLB] to select Endpoint again.
```

Comment:
We may not have to add more information now as we are close to release time, but is this in case of an inconsistency between the OVS groups used for Endpoint section and this table?

Reply:
> but is this in case of an inconsistency between the OVS groups used for Endpoint section and this table?

I guess this flow should be rarely used, unless the case you mentioned. Furthermore, it seems that this flows almost exists with AntreaProxy. I am not sure if there are other reasons for this flow.